### DISCUSSION TITLE
Package name

### DISCUSSION THREAD

**Discussion by soriadelva** [inbo/ClimateCastR#28] [Polls]
Recently I got the question as to whether we'd like to keep the capitalization in the name ClimateCastR. 
There is no real rule that says how a package name should be written, so it's mainly a question of which spelling would be easiest for the user.

**Comment by timadriaens** [comment]
fyi, the other package is spelled climatchR ([paper](https://www.sciencedirect.com/science/article/abs/pii/S1364815222002080))
